duckduckgo/iOS
# Performance Improvement: Caching Easylist Output

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/431340016793878
CC:

**Description**:

Improve performance of page loading times by injected a cached version of the parsed Easylist objects.

**Steps to test this PR**:
1. Ensure pages load correctly
2. Ensure site rating is updated correctly (e.g. using Evans Cycles you should see entries from Disconnect, Easylist and Easylist privacy in app the logs)
3. Ensure blocking occurs in the share extension
4. Ensure whitelisting works 
5. Ensure disabling content blocking works


###### Reviewer Checklist:


###### PR DRI Checklist:
